src/OrderFormAcceptanceTests.Tests/Features/PassPurchasingData.feature

```diff
@@ -0,0 +1,10 @@
+﻿Feature: Pass an Order Form's Purchasing Data to Finance System
+	As an Authority User
+	I want an Order Form's Purchasing Data to be passed to the Finance System for invoicing
+	So that invoices are issued
+@ignore
+Scenario: Pass an Order Form's Purchasing Data
+	Given that the User is on the confirm complete order screen with Funding Source option 'yes' selected
+	When the User confirms to complete the Order
+	Then a .CSV is sent to the specified mailbox
+	And the .CSV to the desired specification is produced (call off-id only)
```

Comment:
Newline at the end of the file to make github happy. I got sick of having to remember this so added the following to the .editorconfig file. This file helps to provide a rule set for coding styles & formatting.

```
# Feature files
[*.feature]
insert_final_newline = true
```

Reply:
I don't see a .editorconfig file in the repo, is this something from VS code? If I was to create one, where should it go so that Visual Studio will pick it up?

Reply:
think I've got it, see later commit